ROMFS/px4fmu_common/init.d/rc.interface

```diff
@@ -11,7 +11,7 @@ then
 	# Load main mixer
 	#
 
-	if [ $MIXER_AUX == none ]
+	if [ $MIXER_AUX == none  -a $USE_IO == yes]
```

Comment:
If there was in fact any issue with FMU-standalone, I would suspect this line here might be the only relevant change.